documentation/README.md

```diff
@@ -0,0 +1,80 @@
+# How we write docs
+
+This guide explains the framework we use for writing documentation for all our projects.
+
+## Classification
+
+We can classify the resource types into four groups:
+
+- Getting started: Provide a quick route to begin using the project; their objective is to teach the basics to get started.
+- User Guide: Provide in-depth walkthroughs to use the project; they are more advanced versions than the "Getting started" guides, but their primary purpose is also teaching.
+- Cookbook: Recipes to achieve a specific goal. They provide short and to-the-point examples to perform common tasks. Their objective is to be a shortcut so the user doesn't have to gather all the pieces from reading the API reference.
```

Comment:
I think cookbook isn't the wide term out there, maybe it should be use cases/how-to/examples/recipes.
https://docs.lakefs.io/